### ISSUE TITLE
Meteor, set default color

### ISSUE THREAD

**Issue by sasikanth513** [mdbootstrap/mdb-ui-kit#346]
@dandv 
 As in original docs

```
@import "../bower_components/bootstrap-material-design/less/material-wfont.less";

// Override @primary color with one took from _colors.less
@primary: @deep-purple;
```

can we set default color in meteor? If so,how to set ?

I'm also using nemo64's bootstrap package, can we set using that package?

**Comment by Obiwarn** [comment]
Same problem here

**Comment by mbabauer** [comment]
I switched to the `twbs:bootstrap` package for bootstrap, but I was wondering the same thing.  I tried doing a file search for `material-wfont.less`, but it was not found.

Next I tried adding `@primary: #673AB7;` to one of my .less files, which is the color I pulled off the [Google Material Color](http://www.google.com/design/spec/style/color.html#color-color-palette) pallet for Deep Purple, but that didn't seem to work either.  I was expecting the following screen element to change colors:

```
<button class="btn btn-primary">Test</button>
```

But it remained the default Teal color.
